Rakefile

```diff
@@ -7,7 +7,7 @@ require 'github_changelog_generator/task'
 GitHubChangelogGenerator::RakeTask.new :changelog do |config|
   config.user = 'riemann'
   config.project = 'riemann-tools'
-  config.exclude_labels = ['skip-changelog']
+  config.exclude_labels = %w[dependencies skip-changelog]
```

Comment:
Dependabot issues are tagged `dependencies`, and we usually de not want them in the CHANGELOG because it has no value for end-users, so exclude this label.

This filter our an older entry in the CHANGELOG that could have been manually excluded previously. Not a big deal IMHO.